package.json

```diff
@@ -18,7 +18,6 @@
   "scripts": {
     "build": "yarn workspaces run build",
     "build:dev": "yarn workspaces run build:dev",
-    "bundlesize": "npx bundlesize",
```

Comment:
do we want to remove this script? We still want to run it if it isn't optional right?

Reply:
Yarn has a feature where [you can run any locally installed CLI with `yarn <cli>` without defining a script.](https://yarnpkg.com/lang/en/docs/cli/#locally-installed-clis) So we don't need this script defined, and now it's not cluttering package.json.